networkmonitor/src/main/res/values/donottranslate_strings_export_html.xml

```diff
@@ -29,66 +29,110 @@ limitations under the License.
 <meta http-equiv=\"Content-Type\" content=\"text/html; charset=utf-8\"/>
 <style type="text/css">
 table {
- border-collapse: collapse;
+ border-spacing: 0px;
```

Comment:
This border-spacing line was part of the solution for the problem with the vertical lines being offset by 1px (in the header vs the body).  Several hours to find that one...